Source/Scene/Camera.js

```diff
@@ -210,7 +212,7 @@ define([
         this._max2Dfrustum = undefined;
 
         // set default view
-        this.viewRectangle(Camera.DEFAULT_VIEW_RECTANGLE, scene.mapProjection.ellipsoid);
+        rectangleCameraPosition3D(this, Camera.DEFAULT_VIEW_RECTANGLE, this.position, true);
```

Comment:
@bagnell using `this.setView` here breaks a whole bunch of the tests and I couldn't figure out why.